pywhat/Data/regex.json

```diff
@@ -1309,6 +1309,23 @@
          ]
       }
    },
+   {
+      "Name": "UPS Tracking Number",
```

Comment:
May I suggest United States Postal Service (UPS) Tracking Number? Our target audience may not be aware that UPS is a thing that exists 😄